addons/l10n_ro_efactura/models/account_move.py

```diff
@@ -0,0 +1,190 @@
+import requests
+
+from odoo import models, fields, _, api
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_ro_edi_document_ids = fields.One2many(
+        comodel_name='l10n_ro_edi.document',
+        inverse_name='invoice_id',
+    )
+    l10n_ro_edi_state = fields.Selection(
+        selection=[
+            ('invoice_sending', 'Sending'),
+            ('invoice_sent', 'Sent'),
+        ],
+        string='E-Factura Status',
+        compute='_compute_l10n_ro_edi_state',
+        store=True,
+    )
+
+    ################################################################################
+    # Compute Methods
+    ################################################################################
+
+    @api.depends('l10n_ro_edi_document_ids', 'l10n_ro_edi_state')
+    def _compute_show_reset_to_draft_button(self):
+        """ Prevent user to reset move to draft when there's an
+            active sending document or an OK response has been received """
+        # EXTENDS 'account'
+        super()._compute_show_reset_to_draft_button()
+        for move in self:
+            received_error_document = (move.l10n_ro_edi_state == 'invoice_sending' and
+                                       all(doc.state != 'invoice_sending_failed' for doc in move.l10n_ro_edi_document_ids))
+            if move.l10n_ro_edi_state == 'invoice_sent' or received_error_document:
```

Comment:
I think you can just do that. (to check with @chklop )
```suggestion
            if move.l10n_ro_edi_state == 'invoice_sent':
```